File: benchmark/2026-09-17-7677581(latest)/standard/rubrics/probes/TEST-1/java/alt/ProbeAssertTests.java
// Supplementary (not the frozen recipe's route): TEST-1 expressed with the JDK's
// built-in `assert` statement, run with `java -ea`. Recorded to test whether an
// admissible first-party assertion construct alone can satisfy V2/V3 for Java.
public class ProbeAssertTests {
    static int add(int a, int b) { return a + b; }

    static void passOne() { assert add(1, 1) == 2 : "pass_one"; }
    static void passTwo() { assert add(2, 3) == 5 : "pass_two"; }
    static void failOne() { assert add(1, 1) == 3 : "fail_one"; }

    public static void main(String[] args) {
        passOne();
        passTwo();
        failOne();
        System.out.println("all assertions held");
    }
}

File: benchmark/2026-09-17-7677581(latest)/standard/rubrics/probes/TEST-1/kotlin/alt/ProbeAssertTests.kt
// Supplementary: TEST-1 expressed with kotlin.test assertion functions only
// (kotlin-test.jar ships in the kotlinc distribution), driven from main(),
// because no JUnit platform runner is installed on this host.
import kotlin.test.assertTrue

fun add(a: Int, b: Int): Int = a + b

fun passOne() { assertTrue(add(1, 1) == 2, "pass_one") }
fun passTwo() { assertTrue(add(2, 3) == 5, "pass_two") }
fun failOne() { assertTrue(add(1, 1) == 3, "fail_one") }

fun main() {
    passOne()
    passTwo()
    failOne()
    println("all assertions held")
}
